kubernetes-manifests/adservice.yaml

```diff
@@ -12,6 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# [START gke_kubernetes_manifests_adservice_deployment_adservice]
```

Comment:
ooooh good point. 
this will cause a problem, since we shouldn't have duplicate region tags (with the same language, in this case yaml)
I'll try to see a workaround, thanks for bringing this up!!

Reply:
I agree. Adding code to remove lines look like `# [START ... ]` and `# [END ...` in [`make-release-artifacts.sh`](https://github.com/GoogleCloudPlatform/microservices-demo/blob/master/hack/make-release-artifacts.sh#L88) might be the best solution.

Reply:
Addressed this! I added 2 lines for the kubernetes and istio manifest section. 
For example [here](https://github.com/GoogleCloudPlatform/microservices-demo/blob/49b5b9c66c87943e72e6e1d4f3684b146551ced5/hack/make-release-artifacts.sh#L96)
```
sed -i '' '/[^# \[]]/d' "${k8s_manifests_file}" 
```
finds lines that start with `# [` and deletes it

Reply:
Awesome! Thanks so much for adding this in. :)

Reply:
after discussion with Megan, I edited [make-release-artifacts.sh](https://github.com/GoogleCloudPlatform/microservices-demo/blob/06e8cdffc6e9fa23bc3acac5657d4e40ad30bac7/hack/make-release-artifacts.sh#L79) to add region tags around the `release/kubernetes-manifests.yaml` and the `release/istio-manifests.yaml`